exonum-java-binding/core/rust/src/runtime/config.rs

```diff
@@ -19,32 +19,21 @@ use std::fmt;
 /// JavaServiceRuntime configuration.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct Config {
-    /// JVM configuration.
-    pub jvm_config: JvmConfig,
-    /// EJB configuration
-    pub ejb_config: EjbConfig,
-    /// Java service configuration.
-    pub service_config: ServiceConfig,
+    /// Private part of the EJB configuration parameters.
+    pub private_config: PrivateConfig,
+    /// Public part of the EJB configuration parameters.
+    pub public_config: PublicConfig,
 }
 
-/// EJB-specific configuration
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct EjbConfig {
-    /// Java service classpath. Must include all its dependencies.
-    ///
-    /// Includes java_bindings internal dependencies as well as user service dependencies.
-    pub class_path: String,
-    /// Path to java-bindings shared library.
-    ///
-    /// Should be path to core/rust/target/{debug, release}
-    pub lib_path: String,
+/// Private EJB configuration.
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+pub struct PrivateConfig {
+    /// Java service classpath.
+    pub service_class_path: String,
     /// Path to `log4j` configuration file.
     pub log_config_path: String,
-}
-
-/// JVM configuration.
-#[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct JvmConfig {
```

Comment:
Do we have to mix several categories of parameters — related to JVM, to service and to the runtime?

Reply:
I will think about splitting them